frontend/src/services/requestService.js

```diff
@@ -0,0 +1,68 @@
+import axios from 'axios';
+
+axios.defaults.headers.common = {
+    'X-Requested-With': 'XMLHttpRequest',
+    'Content-type': 'application/json',
+    'Accept': 'application/json'
+}
+
+const csrfToken = document.head.querySelector('meta[name="csrf-token"]');
+
+if (csrfToken) {
+    axios.defaults.headers.common['X-CSRF-TOKEN'] = csrfToken.content;
+} else {
+    console.error('CSRF token not found: https://laravel.com/docs/csrf#csrf-x-csrf-token');
+}
+
+const token = 'getTokenFromAuthService';
+
+const authHeader = {
+    'Authorization': 'Bearer ' + token
+};
+
+const get = (url, params = {}, headers = {}) => {
+    axios.get(url, {
+        params: params,
+        headers: {
+            ...headers,
+            ...authHeader,
+        }
+    });
+};
+
+const create = (url, data, headers = {}) => {
+    axios.post(url, data, {
+        headers: {
+            ...headers,
+            ...authHeader
+        }
+    });
+};
+
+const update = (url, data, headers = {}) => {
+    axios.put(url, {
+    	data: data,
+        headers: {
+            ...headers,
+            ...authHeader
+        }
+    });
+};
+
+const delete = (url, params = {}) => {
```

Comment:
how did you manage to name variable "delete"? it is the reserved word isn't it?

Reply:
oops. How may be better: destroy or remove?

Reply:
variable you can name as you wish, I mean destroy or deleteRequest. And then name key in the service as "delete", something like this
const requestDelete = () => {/* ...do stuff */}
export default {
    delete: requestDelete
};

but pay attention, according to airbnb style guide keys with values should be put on the top of an object, so if you have an update method, it will look like

export default {
    delete: requestDelete
    update
}

Reply:
destroy is also okay =)